api/services/ProxyDataSync.js

```diff
@@ -1,10 +1,15 @@
 // use dynamodb helper to write items to dynamodb
+const url = require('url');
 const { DynamoDBDocumentHelper } = require('./DynamoDBDocumentHelper');
 const config = require('../../config');
 
-const tableName = config.app.proxySiteTable;
-const siteKey = 'Id';
+const TABLE_NAME = config.app.proxySiteTable;
+const SITE_KEY = 'Id';
 const getSiteKey = site => site.subdomain;
+const getBranchKey = (build) => {
```

Comment:
in terms of performance or simplicity in the proxy, would there be any value in using the `subdomain` and `path` or something that re-uses the key we use for the Site?

Reply:
as discussed ... for now, we will continue with build url origna nd path fur use by origin response